Update aggregates.md (#4235)
duckdb/duckdb-web: docs/sql/functions/aggregates.md

| [`fsum(arg)`](#fsumarg) | Calculates the sum using a more accurate floating point summation (Kahan Sum). |
| [`geomean(arg)`](#geomeanarg) | Calculates the geometric mean of all non-null values in `arg`. |
| [`histogram(arg)`](#histogramarg) | Returns a `MAP` of key-value pairs representing buckets and counts. |
| [`histogram(arg, boundaries)`](#histogramargboundaries) | Returns a `MAP` of key-value pairs representing bucket boundaries and counts of elements in the corresponding left-open and right-closed partition of the datatype. A boundary at the largest value of the datatype is automatically added when elements larger than the largest boundary appear, see [`is_histogram_other_bin`]({% link docs/sql/functions/utility.md %}#ishistogramotherbinarg). |

> | [`histogram_exact(arg, elements)`](#histogramexactargelements) | Returns a `MAP` of key-value pairs representing the requested elements and their counts. A catch-all element specific to the data-type is automatically added to count other elements when they appear, see [`is_histogram_other_bin`]({% link docs/sql/functions/utility.md %}#ishistogramotherbinarg). |

Author: ```suggestion
| [`histogram_exact(arg, elements)`](#histogram_exactargelements) | Returns a `MAP` of key-value pairs representing the requested elements and their counts. A catch-all element specific to the data-type is automatically added to count other elements when they appear, see [`is_histogram_other_bin`]({% link docs/sql/functions/utility.md %}#is_histogram_other_binarg). |
```

| **Example** | `histogram(A)` |
| **Alias(es)** | - |

#### `histogram(arg, boundaries)`

<div class="nostroke_table"></div>

> | **Description** | Returns a `MAP` of key-value pairs representing bucket boundaries and counts of elements in the corresponding left-open and right-closed partition of the datatype. A boundary at the largest value of the datatype is automatically added when elements larger than the largest boundary appear, see [`is_histogram_other_bin`]({% link docs/sql/functions/utility.md %}#ishistogramotherbinarg).  |

Author: ```suggestion
| **Description** | Returns a `MAP` of key-value pairs representing bucket boundaries and counts of elements in the corresponding left-open and right-closed partition of the datatype. A boundary at the largest value of the datatype is automatically added when elements larger than the largest boundary appear, see [`is_histogram_other_bin`]({% link docs/sql/functions/utility.md %}#is_histogram_other_binarg).  |
```

| **Example** | `histogram(A, [0, 1, 10])` |
| **Alias(es)** | - |

#### `histogram_exact(arg, elements)`

<div class="nostroke_table"></div>

> | **Description** | Returns a `MAP` of key-value pairs representing the requested elements and their counts. A catch-all element specific to the data-type is automatically added to count other elements when they appear, see [`is_histogram_other_bin`]({% link docs/sql/functions/utility.md %}#ishistogramotherbinarg). |

Author: ```suggestion
| **Description** | Returns a `MAP` of key-value pairs representing the requested elements and their counts. A catch-all element specific to the data-type is automatically added to count other elements when they appear, see [`is_histogram_other_bin`]({% link docs/sql/functions/utility.md %}#is_histogram_other_binarg). |
```
